### ISSUE TITLE
Offline exporting fails when boost is active

### ISSUE THREAD

**Issue by PadWorks** [highcharts/highcharts#19253]
#### Expected behaviour
When exporting PDF offline, I expect a file with the same data as the displayed trend.

#### Actual behaviour
Once the Boost module is active, the exported PDF-chart is empty. If the boost module is not active, the export basically works. However, if a PDF is exported, an error message appears in the Chrome log for internal parameters that were not found.

#### Live demo with steps to reproduce
Standard: https://jsfiddle.net/PadWorks/stn6w9a7/26/
Boost Active: https://jsfiddle.net/PadWorks/759csfgk/9/

#### Product version
Highcharts JS v11.1.0

#### Affected browser(s)
Chrome

**Comment by karolkolodziej** [comment]
Hi @PadWorks!

It looks like it is a regression introduced in the v9.3.2 probably along with the AST- [demo with the last working version.](https://jsfiddle.net/BlackLabel/mgtcdo7v/)
Internal note: It fails in this loop, the `href` is missing:
https://github.com/highcharts/highcharts/blob/245dfbbcc5b44f5a1e7aea38759be93f324a1d6b/ts/Extensions/OfflineExporting/OfflineExporting.ts#L811-L839

**Comment by PadWorks** [comment]
@jakubSzuminski Does that mean you are about to solve the problem? By when can I expect a fix?

**Comment by jakubSzuminski** [comment]
Hi @PadWorks, I'm currently debugging this issue.

 I cannot give you a specific ETA, but I've recently started working on this problem. I will do everything I can to solve it as soon as possible. I cannot give you any guarantees, but my estimate would be around 2 weeks. 

If I find any simple workaround available, I'll let you know here.

**Comment by jakubSzuminski** [comment]
_Internal note_
It seems that the problem is not just about **boost**, but overall about images in offline exporting: https://jsfiddle.net/BlackLabel/b96q7hof/

**Comment by PadWorks** [comment]
@jakubSzuminski Many thanks. Basically the export now works with image types PNG and JPG, but unfortunately there seems to be a problem with the PDF export. 
1. The file size is not plausible with over 120MB for 12 points? (Above a certain size the exporter dies with activated boost so that the PDF becomes corrupt and is downloaded with 0 Bytes)
2. Highcharts throws warning #33 due to the internal tagName fedropshadow. This should not happen in my opinion.

https://jsfiddle.net/PadWorks/3vz7p18s/

**Comment by jakubSzuminski** [comment]
@PadWorks no problem!

1. Actually, if you take a look at how it used to work in v9.3.1 you'll notice that the file size did not increase due to my fix as it was also above 120MB. The reason for this big size is the high `exporting.zoom` (https://api.highcharts.com/highcharts/exporting.scale) and high resolution.

2. You're right - I've fixed this by allowing the `fedropshadow` tag: https://jsfiddle.net/BlackLabel/9eyf4chm/

This fix does not touch anything related to the file size of the exported file. The only purpose of the solution that I've included is to enable you to export to png/jpeg/pdf with _offline exporting_ the same way you used to be able to do with versions <9.3.2.

I'll look into the PDF corruption you described deeper this week and see if there's anything we can do about this.
